remoteworkitem/trackeritem_repository.go

```diff
@@ -1,10 +1,70 @@
 package remoteworkitem
 
-import "github.com/jinzhu/gorm"
+import (
+	"fmt"
 
-// upload imports the items into database
+	"golang.org/x/net/context"
+
+	"github.com/almighty/almighty-core/app"
+	"github.com/almighty/almighty-core/models"
+	query "github.com/almighty/almighty-core/query/simple"
+	"github.com/jinzhu/gorm"
+)
+
+// Upload imports the items into database
 func upload(db *gorm.DB, tqID int, item map[string]string) error {
 	ti := TrackerItem{Item: item["content"], RemoteItemID: item["id"], BatchID: item["batch_id"], TrackerQueryID: uint64(tqID)}
 	err := db.Create(&ti).Error
 	return err
 }
+
+// Map a remote work item into an ALM work item and persist it into the database.
+func convert(ts *models.GormTransactionSupport, tqID int, item map[string]string, provider string) (*app.WorkItem, error) {
+	witr := models.NewWorkItemTypeRepository(ts)
+	wir := models.NewWorkItemRepository(ts, witr)
+	ti := TrackerItem{Item: item["content"], RemoteItemID: item["id"], BatchID: item["batch_id"], TrackerQueryID: uint64(tqID)}
+
+	// Converting the remote item to a local work item
+	remoteTrackerItemMethodRef := RemoteWorkItemImplRegistry[provider]
+	remoteTrackerItem, err := remoteTrackerItemMethodRef(ti)
+	if err != nil {
+		return nil, BadParameterError{parameter: provider, value: nil}
+	}
+	workItem, err := Map(remoteTrackerItem, WorkItemKeyMaps[provider])
+	if err != nil {
+		return nil, ConversionError{simpleError{message: "Error mapping to local work item "}}
+	}
+	fmt.Println(workItem.Fields)
```

Comment:
Use `log` package for logging. If this is just for debugging, you can remove it.